experimental/packages/opentelemetry-sdk-metrics-base/src/state/MetricCollector.ts

```diff
@@ -44,14 +44,14 @@ export class MetricCollector implements MetricProducer {
    * Delegates for MetricReader.forceFlush.
    */
   async forceFlush(): Promise<void> {
-    return this._metricReader.forceFlush();
+    await this._metricReader.forceFlush({});
```

Comment:
Think the options object is optional now

Reply:
True. Addressed it in 056ea19. Thanks!